iroh-rpc-client/Cargo.toml

```diff
@@ -2,41 +2,30 @@
 name = "iroh-rpc-client"
 version = "0.1.3"
 edition = "2021"
-authors = ["dignifiedquire <me@dignifiedquire.com>", "ramfox"]
+authors = ["Friedel Ziegelmayer <me@dignifiedquire.com>", "ramfox"]
 license = "Apache-2.0/MIT"
 repository = "https://github.com/n0-computer/iroh"
 description = "RPC type client for iroh"
-rust-version = "1.63"
+rust-version = "1.65"
 
 [dependencies]
 
 anyhow = { version = "1", features = ["backtrace"] }
 async-stream = "0.3.3"
-async-trait = "0.1.56"
 bytes = "1.1.0"
-cid = "0.9"
+cid = "0.9.0"
 config = "0.13.1"
 futures = "0.3.21"
-iroh-metrics = { version = "0.1.3", path = "../iroh-metrics", default-features = false }
-iroh-rpc-types = { version = "0.1.3", path = "../iroh-rpc-types", default-features = false }
-iroh-util = { version = "0.1.3", path = "../iroh-util" }
+iroh-metrics = { path = "../iroh-metrics", default-features = false }
```

Comment:
Will need `version` to be able to be published I think?
(also other entries)

Reply:
yep